armhf/gtk+2.0-maemo-2.24.31-r0.apk (needed for hildon-desktop) build fails

[23:18:02] (buildroot_armhf) install autoconf automake libtool atk-dev cairo-dev intltool libxdamage-dev pango-dev cups-dev expat-dev gdk-pixbuf-dev gnutls-dev gobject-introspection-dev gtk-doc hicolor-icon-theme libice-dev libx11-dev libxcomposite-dev libxcursor-dev libxext-dev libxfixes-dev libxrandr-dev libxi-dev
[23:18:26] (buildroot_armhf) build armhf/gtk+2.0-maemo-2.24.31-r0.apk
 [00:01:29] NOTE: The failed command's output is above the ^^^ line in the logfile: /data/tmp/pmbootstrap/log.txt
[00:01:29] ERROR: Command failed: (buildroot_armhf) % cd /home/user/build && su user -c 'CARCH=armhf PATH=/usr/lib/distcc/bin:/usr/lib/ccache/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin DISTCC_HOSTS=127.0.0.1:33632 abuild -d'
[00:01:29] Run 'pmbootstrap log' for details.
[00:01:29] See also: <https://postmarketos.org/troubleshooting>
pavel@duo:/data/l/pmos/pmbootstrap$  df -h
Filesystem      Size  Used Avail Use% Mounted on
/dev/root        14G  9.4G  3.8G  72% /
devtmpfs        2.9G     0  2.9G   0% /dev
tmpfs           2.9G  364M  2.5G  13% /dev/shm
tmpfs           2.9G   18M  2.9G   1% /run
tmpfs           5.0M  8.0K  5.0M   1% /run/lock
tmpfs           2.9G     0  2.9G   0% /sys/fs/cgroup
/dev/sda2       429G  420G  9.7G  98% /data
tmpfs           584M  4.0K  584M   1% /run/user/107
tmpfs           584M   36K  584M   1% /run/user/1000
pavel@duo:/data/l/pmos/pmbootstrap$ ./pmbootstrap.py log
    from gtkdoc import common, config, rebase
  File "/usr/share/gtk-doc/python/gtkdoc/rebase.py", line 28, in <module>
    from six import iteritems, iterkeys
ImportError: No module named six
make[4]: *** [Makefile:1031: install-data-local] Error 1
make[4]: Leaving directory '/home/user/build/src/gtk+-2.24.31/docs/reference/gdk'
make[3]: *** [Makefile:717: install-am] Error 2
make[3]: Leaving directory '/home/user/build/src/gtk+-2.24.31/docs/reference/gdk'
make[2]: *** [Makefile:505: install-recursive] Error 1
make[2]: Leaving directory '/home/user/build/src/gtk+-2.24.31/docs/reference'
make[1]: *** [Makefile:507: install-recursive] Error 1
make[1]: Leaving directory '/home/user/build/src/gtk+-2.24.31/docs'
make: *** [Makefile:733: install-recursive] Error 1
>>> ERROR: gtk+2.0-maemo*: package failed
>>> ERROR: gtk+2.0-maemo: all failed
(022325) [00:01:29] ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
(022325) [00:01:29] NOTE: The failed command's output is above the ^^^ line in the logfile: /data/tmp/pmbootstrap/log.txt
(022325) [00:01:29] ERROR: Command failed: (buildroot_armhf) % cd /home/user/build && su user -c 'CARCH=armhf PATH=/usr/lib/distcc/bin:/usr/lib/ccache/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin DISTCC_HOSTS=127.0.0.1:33632 abuild -d'
(022325) [00:01:29] Run 'pmbootstrap log' for details.

I got the same (or similar?) error:

Traceback (most recent call last):
  File "/usr/bin/gtkdoc-rebase", line 28, in <module>
      from gtkdoc import common, config, rebase
        File "/usr/share/gtk-doc/python/gtkdoc/rebase.py", line 28, in
  <module>
      from six import iteritems, iterkeys
      ImportError: No module named six
      make[4]: *** [Makefile:1031: install-data-local] Error 1
      make[4]: Leaving directory
  '/home/user/build/src/gtk+-2.24.31/docs/reference/gdk'

I now added "py-six" package, too. Lets see what happens...

--- a/aports/maemo/gtk+2.0-maemo/APKBUILD
+++ b/aports/maemo/gtk+2.0-maemo/APKBUILD
@@ -1,6 +1,6 @@
 pkgname=gtk+2.0-maemo
 pkgver=2.24.31
-pkgrel=0
+pkgrel=1
 subpkg=gtk-update-icon-cache
 pkgdesc="The GTK+ Toolkit (v2)"
 url="http://www.gtk.org/"
@@ -43,7 +43,8 @@ makedepends="
 	libxfixes-dev
 	libxrandr-dev
 	libxi-dev
-	zlib-dev"
+	zlib-dev
+	py-six"
 source="http://ftp.gnome.org/pub/gnome/sources/gtk+/${pkgver%.*}/gtk+-$pkgver.tar.xz
 	https://raw.githubusercontent.com/fremantle-gtk2/gtk/2.24.25-3m7%2Bmaemo/debian/patches/hildonize-gdk-window.diff
 	https://raw.githubusercontent.com/fremantle-gtk2/gtk/2.24.25-3m7%2Bmaemo/debian/patches/hildonize-gtk-container.diff

